supplementary_style_guide/glossary_terms_conventions/general_conventions/c.adoc

```diff
@@ -601,6 +601,18 @@ In IdM, you can also create multiple *sub-CAs*. Sub-CAs are IdM CAs whose certif
 
 *See also*: xref:content-view[Content View]
 
+discrete]
+[[compute-node]]
+==== image:images/yes.png[yes] compute node (noun)
```

Comment:
This entry isn't alphabetical. It needs to go below "composite role"

Reply:
Compute node is a common term and component in OpenStack also, so maybe the entry could be changed to be more multiple-product friendly. 

I guess the concept is the same since they're the worker nodes (but in OSP there's no "lucter users").